src/website/shared/models/cve.py

```diff
@@ -293,13 +295,13 @@ class NixpkgsIssue(models.Model):
     created = models.DateField(auto_now_add=True)
     code = models.CharField(max_length=len("NIXPKGS-YYYY-") + 19)
 
-    cve = models.ManyToManyField(CveRecord)
```

Comment:
I don't understand what is the value of this change, also it seems unrelated to that PR